packages/react-percy/src/cli.js

```diff
@@ -1,15 +1,16 @@
 import * as args from './args';
 import chalk from 'chalk';
-import pkgDir from 'pkg-dir';
 import readPercyConfig from '@percy-io/react-percy-config';
 import { resolve as readWebpackConfig } from '@percy-io/react-percy-webpack';
 import runPercy from '@percy-io/react-percy-ci';
 import yargs from 'yargs';
 
+process.env.NODE_ENV = process.env.NODE_ENV || 'development';
+
 const VERSION = require('../package.json').version;
 
 // eslint-disable-next-line import/prefer-default-export
-export function run(argv) {
+export function run(argv, rootDir) {
```

Comment:
This is just for testing purposes. Allows setting the project root directory to something other than the process's current working directory.